docassemble/MotionToStayEviction/data/questions/review_screen.yml

```diff
@@ -0,0 +1,386 @@
+---
+event: review_case_info
+id: |-
+  review screen
+question: |-
+  Review your answers
+review:
+  - Edit: users.revisit
+    button: |
+      **Your party: (Edit to change name and address)**
+
+      %  for my_var in users:
+        * ${ my_var.name }
+      % endfor
+  - Edit: other_parties.revisit
+    button: |
+      **The other party: (Edit to change name and address)**
+      
+      % for my_var in other_parties:
+        * ${ my_var.name }
+      % endfor
+  - Edit: users[0].address.address
+    button: |
+      **Your address:**
+      ${ users[0].address.on_on_line() }
+  - Edit: |-
+      case_new_or_existing
+    button: |
+      **Is this a new case?**
+
+      ${ showifdef('case_new_or_existing') }
+  - Edit: |-
+      users.there_is_another
+    button: |
+      **Did anyone else file with you?**
+
+      Is anyone else on your side of the case?: ${ word(yesno(users.there_is_another)) }
+  - Edit: |-
+      public_assistance_kinds
```

Comment:
Editing this is fine, but you can edit out the fact that you don't get any benefits, and we don't ask the follow up questions.

Review screens always seem to open up these types of rabbit holes that break most of the interview logic, and I'm never sure how far to try to fix them.